features/step_definitions/destroy_steps.rb

```diff
@@ -0,0 +1,7 @@
+Then("I click on the {string} link") do |string|
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+Given("I am on the {string} page") do |string|
```

Comment:
Is this step actually used somewhere? Also, you're capturing a `string` here which I assume is a `page name` but you're not doing anything with it.

You might as well just have a step that says
`Given I am on the inbox page` rather than `Given I am on the "inbox" page`

If you create the latter you can actually set the body of that step such that it handles multiple pages on your page. The logic would decide which page to visit based on the `page_name`  that's extracted from the scenario's step